sources/osgUtil/IntersectionVisitor.js

```diff
@@ -156,7 +156,7 @@ IntersectionVisitor.prototype = MACROUTILS.objectInherit( NodeVisitor.prototype,
 
     applyNode: function ( node ) {
         if ( !this.enter( node ) ) return;
-        if ( node.primitives ) {
+        if ( node.getPrimitiveSetList ) {
```

Comment:
my bad it's faster to check property https://jsperf.com/instanceof-performance/25